CHORE: update kvv2 patch docs to clarify behavior when secret does not exist (#1008)
hashicorp/web-unified-docs: content/vault/v1.20.x/content/docs/secrets/kv/kv-v2/cookbook/patch-data.mdx

Reviewer: ```suggestion
- You know the keys or [subkeys](/vault/docs/secrets/kv/kv-v2/cookbook/read-subkey)
you want to patch.
- The secret path you want to update already exists. You cannot create
  new paths with a patch.
```

<Tip title="Assumptions">

- You have [set up a `kv` v2 plugin](/vault/docs/secrets/kv/kv-v2/setup).
- Your authentication token has appropriate permissions for the `kv` v2 plugin:
   - **`patch`** permission to make direct updates with `PATCH` actions.
   - **`create`**+**`update`** permission if you want to make indirect
      updates with the Vault CLI by combining `GET` and `POST` actions.
- You know the keys or [subkeys](/vault/docs/secrets/kv/kv-v2/cookbook/read-subkey)

> you want to patch.

Reviewer: ```suggestion
  you want to patch.
```

</Tip>

> Note: if the secret does not already exist, the patch will not create it.

Reviewer: ```suggestion
- The keys or subkeys already exist with editable values. Patching does
   not create new secrets.
```

Instead of having a floating sentence, we should add this to the list after "You know the keys or subkeys you want to patch".
